Mathlib/Analysis/Normed/Lp/ProdLp.lean

```diff
@@ -402,6 +402,13 @@ theorem prod_lipschitzWith_equiv_aux [PseudoEMetricSpace α] [PseudoEMetricSpace
           gcongr
           simp only [self_le_add_left]
 
+theorem prod_lipschitzWith_equiv_aux [PseudoEMetricSpace α] [PseudoEMetricSpace β] :
+    LipschitzWith 1 (WithLp.equiv p (α × β)) := by
```

Comment:
I think you can golf this with `.of_edist_le`, but probably too minor to justify restarting bors